streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -88,23 +88,25 @@ public void onPartitionsAssigned(Consumer<?, ?> consumer, Collection<TopicPartit
 
         @Override
         public void onPartitionsRevoked(Consumer<?, ?> consumer, Collection<TopicPartition> assignment) {
-            commitAll(time.milliseconds());
+            commitAll();
             removePartitions();
             lastClean = Long.MAX_VALUE; // stop the cleaning cycle until partitions are assigned
         }
     };
 
-    public StreamThread(TopologyBuilder builder, StreamingConfig config) throws Exception {
-        this(builder, config, null , null, new SystemTime());
+    public StreamThread(int threadId, TopologyBuilder builder, StreamingConfig config) throws Exception {
+        this(threadId, builder, config, null , null, new SystemTime());
     }
 
     @SuppressWarnings("unchecked")
-    StreamThread(TopologyBuilder builder, StreamingConfig config,
+    StreamThread(int threadId,
+                 TopologyBuilder builder, StreamingConfig config,
                  Producer<byte[], byte[]> producer,
                  Consumer<byte[], byte[]> consumer,
                  Time time) throws Exception {
-        super("StreamThread-" + nextThreadNumber.getAndIncrement());
+        super("StreamThread-" + threadId);
```

Comment:
What happens to the thread names when multiple KafkaStreaming instances are running in the same JVM?